docs/development/build-system-overview.md

```diff
@@ -1,71 +1,72 @@
+will be linked to a shared library version of Chromium's components to achieve
 fast linking time; for `Release` builds, the binary will be linked to the static
 library versions, so we can have the best possible binary size and performance.
 
-## Minimal bootstrapping
+## Minimal Bootstrapping
 
-All of Chromium's prebuilt binaries(libchromiumcontent) are downloaded when
+All of Chromium's prebuilt binaries (`libchromiumcontent`) are downloaded when
 running the bootstrap script. By default both static libraries and shared
 libraries will be downloaded and the final size should be between 800MB and 2GB
-according to the platform.
+depending on the platform.
 
-By default, libchromiumcontent is downloaded from Amazon Web Services.
-If the `LIBCHROMIUMCONTENT_MIRROR` environment variable is set, bootrstrap
+By default, `libchromiumcontent` is downloaded from Amazon Web Services.
+If the `LIBCHROMIUMCONTENT_MIRROR` environment variable is set, the bootstrap
 script will download from it.
-[libchromiumcontent-qiniu-mirror](https://github.com/hokein/libchromiumcontent-qiniu-mirror)
-is a mirror for libchromiumcontent. If you have trouble in accessing AWS, you can
-switch download address to it via `export LIBCHROMIUMCONTENT_MIRROR=http://7xk3d2.dl1.z0.glb.clouddn.com/`
+[`libchromiumcontent-qiniu-mirror`](https://github.com/hokein/libchromiumcontent-qiniu-mirror)
+is a mirror for `libchromiumcontent`. If you have trouble in accessing AWS, you
+can switch the download address to it via
+`export LIBCHROMIUMCONTENT_MIRROR=http://7xk3d2.dl1.z0.glb.clouddn.com/`
 
 If you only want to build Electron quickly for testing or development, you
-can only download the shared library versions by passing the `--dev` parameter:
+can download just the shared library versions by passing the `--dev` parameter:
 
 ```bash
 $ ./script/bootstrap.py --dev
 $ ./script/build.py -c D
 ```
 
-## Two-phrase project generation
+## Two-Phrase Project Generation
 
 Electron links with different sets of libraries in `Release` and `Debug`
-builds, however `gyp` doesn't support configuring different link settings for
+builds. `gyp`, however, doesn't support configuring different link settings for
 different configurations.
 
 To work around this Electron uses a `gyp` variable
-`libchromiumcontent_component` to control which link settings to use, and only
+`libchromiumcontent_component` to control which link settings to use and only
 generates one target when running `gyp`.
 
-## Target names
+## Target Names
 
 Unlike most projects that use `Release` and `Debug` as target names, Electron
 uses `R` and `D` instead. This is because `gyp` randomly crashes if there is
-only one `Release` or `Debug` build configuration is defined, and Electron has
-to only generate one target for one time as stated above.
+only one `Release` or `Debug` build configuration defined, and Electron only has
+to generate one target at a time as stated above.
```

Comment:
I want to double check here as well that I haven't changed the intended meaning @zcbenz